### ISSUE TITLE
[plot] curves on right axis are not selectable

### ISSUE THREAD

**Issue by t20100** [silx-kit/silx#1574]
With matplotlib backend, curves on right axis cannot be selected with the mouse.

To reproduce:
```
from silx import sx
w = sx.plot((1, 2))
w.addCurve((0, 2), (1, 2), legend='2', yaxis='right')
```
and click on the red curve

**Comment by vallsv** [comment]
+1

It is a feedback from bliss/flint user. As this application uses a lot both axis.

**Comment by t20100** [comment]
closed by #2602
